### ISSUE TITLE
TensorBoard just opens a blank webpage (windows)

### ISSUE THREAD

**Issue by georgeparry12** [tensorflow/tensorboard#3496]
When I run my script it trains fine and I can see the "logs" directory appear in my directory. 

However when I go run:  

`tensorboard --logdir=logs/fit`

 The link provided just produces a blank webpage. 
I have also tried various interactions of this including 
```
tensorboard --logdir=logs/fit --bind_all
tensorboard --logdir=logs\fit
```
and other similar adjustments 

If I try and edit the "logs" directory in any way I get an error message saying that it's in use in another program, this does not happen when I have not run the above code. 

I'd assume that means TensorBoard is using it... but I'm unsure...

```
log_dir = "logs\\fit\\" + datetime.datetime.now().strftime("%Y%m%d-%H%M%S")
tensorboard = TensorBoard(log_dir=log_dir, histogram_freq=1)

# train the network
print("[INFO] training network...")
H = model.fit_generator(
	aug.flow(trainX, trainY, batch_size=BS),
	validation_data=(testX, testY),
	steps_per_epoch=len(trainX) // BS,
	callbacks=[tensorboard],
	epochs=EPOCHS, verbose=1)
  
```

![Screenshot 2020-04-08 at 5 27 16 pm](https://user-images.githubusercontent.com/61587443/78809039-3baa8700-79be-11ea-9471-07a47ba81638.png)

I'm running on tensorboard 2.1.0. 

Please let me know if any further information would be useful

**Comment by psybuzz** [comment]
Thanks for the report.  This looks similar to issue https://github.com/tensorflow/tensorboard/issues/3117 affecting many Windows users, which was fixed by https://github.com/tensorflow/tensorboard/pull/3128 in TensorBoard 2.2.0.

Could you please install TensorBoard 2.2.0 or higher, and let us know if it's still blank.

**Comment by georgeparry12** [comment]
Hi psybuzz, thank you for your reply. 

After several attempts, I couldn't get TensorBoard 2.2.0 to install on top of my TensorFlow GPU 2.1.0. It either would not upgrade past TensorBoard 2.1.0 or seemed to mess up the environment when it did install, preventing me from using TensorFlow or TensorBoard. 

I have now got one environment with TensorFlow and everything else installed, and a separate environment with just TensorBoard 2.2.0. Doing it this way now allows me to open up TensorBoard in the web browser. I get this error message upon opening TensorBoard: 
"TensorFlow installation not found - running with reduced feature set." 

But from a first impression, it "seems" fine/loads. 

I was wondering if you had any solutions around this weird set up/if this set up is going to cause me issues?

**Comment by tovi-dev** [comment]
Hi everybody,
I am facing the same issue of a blank result when using the tensorboard command inside a virtual jupyter notebook.
I am working on a Google Cloud Virtual Machine: these are the specifics of the VM:
Machine type: n1-standard-8 (8 vCPU, 30 GB RAM)
GPU: 8 x NVIDIA Tesla K80
Note that. even though the VM has a GPU, I am not using it right now, but I might need it in the future. Therefore what I would like is a solution that potentially could work both with tensorflow and tensorflow-gpu. 

Operating system: Debian GNU/Linux 9 (stretch)

Virtual environment creation: using conda
>>> conda create -n venv python=3.6 anaconda
Installation:
>>> conda activate venv
I specify the latest stable version of tensorflow.
>>> conda install -c conda-forge tensorflow==2.1

<img width="449" alt="conda_tf_pckgs" src="https://user-images.githubusercontent.com/63409624/78904095-5ba08000-7a7c-11ea-9e88-05dc96ba8e17.PNG">

When I try to install tensorboard 2.2, either with conda install or conda-forge, I get:

conda install -c conda-forge tensorboard==2.2
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
PackagesNotFoundError: The following packages are not available from current channels:
 - tensorboard==2.2

So I’m stuck with tensorboard==2.1.

When I launch the virtual jupyter notebook and try some toy code using MNIST dataset, here’s what happens: I am following the documentation at: https://www.tensorflow.org/tensorboard/tensorboard_in_notebooks  
The code is the same, I’m just using 3 epochs instead of 5.

<img width="433" alt="toy_example_jupyter_train" src="https://user-images.githubusercontent.com/63409624/78904189-783cb800-7a7c-11ea-9f2c-0428c048fe5b.PNG">

When I launch the magic command:
>>> %tensorboard --logdir logs 
I get the blank result, saying: xx.xx.xxx.xx took too much time to respond.

![server_resp](https://user-images.githubusercontent.com/63409624/78904231-825eb680-7a7c-11ea-998b-93294ec8560d.png)

Any help would be very appreciated! Please let me know if you need additional info to debug or reproduce this.

**Comment by rohangpatil** [comment]
Happening with Firefox on Mac as well. The screenshot of the error in the console attached. Tensorboard 2.2.0
![Screen Shot 2020-04-13 at 12 53 19 PM](https://user-images.githubusercontent.com/1222850/79155633-454d3980-7d86-11ea-8894-be0065f8a2f1.png)

**Comment by psybuzz** [comment]
Looking at Anaconda [package lists](https://docs.anaconda.com/anaconda/packages/pkg-docs/), it seems none currently have TensorBoard 2.2.  If conda is a requirement, I don't have a recommendation there, other than requesting from the Anaconda team to upgrade their package lists to include TensorBoard 2.2 or higher.

Since the root cause with this issue is with OS mime type settings interfering with TensorBoard, other Github issues have mentioned workarounds.  By modifying your system registry at your own risk, it may be possible to resolve the issue without installing a new TensorBoard.  For example,
https://github.com/tensorflow/tensorboard/issues/2771#issuecomment-569810749
https://github.com/tensorflow/tensorboard/issues/3117#issuecomment-572274516


Thanks for the screenshot, @rohangpatil.  I'm unable to reproduce this error with Firefox on Mac, it looks like that needs some investigation.  If we presume it's the same root cause as on Windows, perhaps there is some setting on your device that is associating mime types?

**Comment by sanatmpa1** [comment]
@georgeparry12,

Please take a look at the above comment from @psybuzz that its fixed in `tensorboard 2.2`. We recommend you to update your tensorboard to latest stable version and reopen this issue or create a new one if the issue still persists. Thanks!
